### DISCUSSION TITLE
Απορία για το About Themes στο βιογραφικό (1)

### DISCUSSION THREAD

**Discussion by AimiliosPavlidis2001** [OompaLoompas/help#2] [Q&A] [answered]
Καλησπέρα ομάδα, το βιογραφικό μου βρίσκεται στα τελικά στάδια ολοκλήρωσής του, όμως το μόνο που έχει μείνει είναι η διαγραφή του About Themes από το sidebar. Όποιος έχει κάτι να προτείνει θα το εκτιμούσα πολύ.

**Comment by p19pasc** [comment, marked as answer]
Καλησπέρα Αιμίλιε, χαίρομαι που είσαι σε καλό δρόμο με το βιογραφικό σου και πιστεύω πως το πρόβλημα σου θα λυθεί άμεσα. Βλέπω πως έχεις επιλέξει το [1ο template](https://github.com/sharu725/online-cv) το οποίο επεξεργάζομαι και εγώ. Μπορείς να μπεις στο data.yml αρχείο σου και να ελέγξεις το block εντολών που  βρίσκεται στην αρχή του `sidebar:` και πριν από το Profile Information.

**Reply by AimiliosPavlidis2001** [reply]
Σε ευχαριστώ Θωμά, ενώ είχα κάνει false το education για να βρίσκεται στο main section, δεν είχα προσέξει από πάνω του πως αν κάνω false το about διαγράφεται το "how to use?".
